### ISSUE TITLE
Correction - Activity 3.6.14

### ISSUE THREAD

**Issue by jehibdon** [TeamBasedInquiryLearning/calculus#152]
We either need to reword or change the function to the one being doing in Activity 3.6.12.

**Comment by StevenClontz** [comment]
Link: [Activity 3.6.12](https://teambasedinquirylearning.github.io/calculus/2024e/AD6.html#activity-derivative-concavity4) 
https://teambasedinquirylearning.github.io/calculus/2024e/AD6.html#activity-derivative-concavity4

**Comment by siwelwerd** [comment]
Every algebraic example in this section (3.6.12--3.6.15) has a critical point at zero, perhaps some more variety would be helpful as well.

**Comment by AbbyANoble** [comment]
Agreed about needing variety. We just did this section last week and I had to pull other examples in. We needed some where f'' had undefined values.

Similar issues in the CheckIt as they are only polynomials for AD5 and AD6.

**Comment by siwelwerd** [comment]
> Similar issues in the CheckIt as they are only polynomials for AD5 and AD6.

Probably would be best to make a separate issue for changes to the CheckIt
